app/src/androidTest/java/testUtils/TestConstants.java

```diff
@@ -0,0 +1,13 @@
+package testUtils;
+
+import testUtils.mockWebServer.MockWebUtils;
+
+public class TestConstants
+{
+
+    public static String VEHICLE_NUMBER = "";
```

Comment:
Does the GPS SDK know what a vehicle number is?

Reply:
Yes, Please refer the screenshot of How consumer app starts services using init module . 
![image](https://user-images.githubusercontent.com/38068550/77508464-c48ad400-6e90-11ea-9433-145585c1ee3a.png)

It is just a value which is to be stored in database. There are no logical checks on this field.